python/ray/serve/deployment_state.py

```diff
@@ -771,7 +771,7 @@ def resource_requirements(self) -> Tuple[str, str]:
         required = {
             k: v
             for k, v in self._actor.actor_resources.items()
-            if v is not None and v > 0
+            if v is not None and isinstance(v, int) and v > 0
```

Comment:
`v` might be a `float` instead of an `int`.

Also, could we can omit this check? Is there any case where `v` isn't a number?